index.html

```diff
@@ -609,8 +634,11 @@ <h2>Queue a <code>PerformanceEntry</code></h2>
             </li>
           </ol>
         </li>
-        <li>If the <var>add to performance entry buffer</var> flag is set, add
-        <var>new entry</var> to the <a>performance entry buffer</a>.
+        <li>
+          Let <var>tuple</var> be the <a>relevant performance entry tuple</a>,
```

Comment:
I think it'll be clearer to split that up into 2 steps. Also, the max buffer limitation should be clearer, to avoid off-by-one errors.

Maybe something like:
* Let tuple be the relevant.. tuple
* If the number of entries in the tuple's performance entry buffer is smaller than tuple's maxBufferSize, append...

In ResourceTiming, we explicitly created variables that count the number of entries, but I think a "number of entries" language would be precise enough

Reply:
Done.  And again I think this should remain here for two reasons: 
1) It is more central information than a note, it is part of the queuing algorithm for a PerformanceEntry
2) It is not re-used anywhere else in the spec to be a common algorithm